source/PowerAdjusterView.mc

```diff
@@ -266,43 +277,36 @@ class PowerDataField extends Ui.DataField {
       power_array_complete = true;
     }
 
-    //Sys.println(POWER_MULTIPLIER);
-    // Sys.println("" + power_sum + "," + avgPower + "," + DURATION + "," + info.altitude);
     var watts = power_sum / (power_array_complete ? DURATION : power_array_next_index);
-    //Sys.println(info.currentPower);
-    //Sys.println(watts);
-    //Sys.println(my_zones);
     powerValue = Math.round(watts).toNumber();
+    
+    if (my_zones.size() > 0 && dc_height != null) {
+      AddPowerToChart(powerValue, ColorMyZone(getPowerZone(powerValue)));
+    }
   }
 
   function onLayout(dc) {
-    setLayout(Rez.Layouts.PowerFieldLayout(dc));
-  }
-
-  function drawZones(dc, zone, power) {
```

Comment:
I'd prefer to keep the old functionality with the color indicator (it's easier to see which zone you in the interval), on top on ability to change to the Graph (in options) if one prefers.